catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -393,6 +404,18 @@ private Double degreesToMeters(double distance) {
                 Distance.LinearUnit.KILOMETER).getAs(Distance.LinearUnit.METER);
     }
 
+    private Integer getSolrQueryRowsProperty() {
+        try {
+            Integer rows = Integer.valueOf(System.getProperty(SOLR_QUERY_ROWS_PROPERTY));
```

Comment:
❓Instead of putting a default value in the `system.properties`, should we  use `getProperty(SOLR_QUERY_ROWS_PROPERTY, SOLR_QUERY_DEFAULT_ROWS)` here? That way we wouldn't have to put anything in the `system.properties` file and document what it means anywhere.